src/TwigFractal.php

```diff
@@ -1,20 +1,48 @@
 <?php
+/**
+ * @file
+ * Contains Drupal\twig_fractal\TwigFractal.
+ */
 
 namespace Drupal\twig_fractal;
 
 use Drupal\twig_fractal\TokenParser\Render;
 use Twig_Extension;
 
+/**
+ * Registers a 'render' tag to make Fractal components reusable in Drupal 8.
+ *
+ * Example usage:
+ * @code
+ * {% render '@foo/baz.twig' %}
+ * {% render '@foo/baz.twig' with { qux: 'foobar' } %}
+ * {% render '@foo/baz--bar.twig' with { qux: 'foobar' } %}
+ * @endcode
+ *
+ * @implements Twig_ExtensionInterface
+ */
```

Comment:
Maybe here should we also refer the render class we already implemented? IMO some class should have to the fractal variants thing project/spects/api/... not sure about which one makes more sense to have it.